### PR TITLE
#2456 talker_listener.launch -> talker_listener_launch

### PR DESCRIPTION
**Pull request by FHefterSRH**
As mentioned in the bug report. This would be the solution for all the poor beginners who struggled with the first examples.

Fixes https://github.com/docker-library/docs/issues/2456

### CODE REVIEW

**Comment by tfoote** [comment]
For better cross reference this is for issue #2456 (bug numbers in the title don't generate automatic cross links, they need to be in the body.)

**Review by tfoote — COMMENTED** [review]
The changes look good to me. I am not sure about whether the changes should be propogated to the README.md or just be in content.md.  As the top of the README says:

https://github.com/docker-library/docs/pull/2458/files#diff-57d89c90a089a624386a955b21854149d6e32cdb49bb2bc9715c8d068b2951d1R5-R12
```
WARNING:
    DO NOT EDIT "ros/README.md"
    IT IS AUTO-GENERATED
(from the other files in "ros/" combined with a set of templates)
```

File: ros/README.md
Comment (whalelines) [inline ros/README.md:1]: As indicated by https://github.com/docker-library/docs/actions/runs/9686952106/job/26734132584?pr=2458 , you should not edit the README.md yourself. Please undo these changes.

**Review by whalelines — COMMENTED** [review]
Please revert the changes to the README

**Comment by FHefterSRH** [comment]
My plan was to do this: "gh checkout origin/master -- ros/README.md"
But I found no way to do it in the web frontend quickly.
So I deleted the file from PR. Hope this is right?

**Comment by whalelines** [comment]
It appears this PR now deletes the README.md rather than undoing the changes you made to it. Perhaps just use the CLI?

**Comment by FHefterSRH** [comment]
In CLI it took seconds. Grr..

**Review by tianon — APPROVED** [review]
Thank you! (both for the patch and for sticking with it to get it correct :heart:)
